src/Agent.Listener/Configuration/AgentConfigSettings.cs

```diff
@@ -0,0 +1,27 @@
+﻿
+namespace Microsoft.VisualStudio.Services.Agent.Listener.Configuration
+{
+
+    public class AgentConfigSettings
+    {
+        public int PoolId;
```

Comment:
dont have any business logic to manipulate on these variable members. Only constructor will set them and then it will be read by consumers